.vib/airflow/vib-verify.json

```
@@ -57,6 +57,12 @@
               "password": "ComplicatedPassword123!4"
             }
           }
+        },
+        {
+          "action_id": "kubescape",
+          "params": {
+            "threshold": {VIB_ENV_KUBESCAPE_SCORE_THRESHOLD}
```

Just for consistency, other env. vars are quoted
```suggestion
            "threshold": "{VIB_ENV_KUBESCAPE_SCORE_THRESHOLD}"
```

---

The missing quotes are in purpose because of variable types.

The `threshold` variable is of type `number`, therefore it requires to be set to:

```
            "threshold": 97.0
```

By adding the quotes, it switches to `string` type, causing an error:
```
            "threshold": "97.0"
```

---

Problem is most linters detect this as an error:
```
"threshold": {VIB_ENV_KUBESCAPE_SCORE_THRESHOLD}
```

But it is not a problem because string substitution happens during the `vmware-image-builder-action` step.

---

For instance, the result would look like this:

```
VIB_ENV_KUBESCAPE_SCORE_THRESHOLD=99.99
"threshold": {VIB_ENV_KUBESCAPE_SCORE_THRESHOLD} -> "threshold": 99.99 (type: number)
"threshold": "{VIB_ENV_KUBESCAPE_SCORE_THRESHOLD}" -> "threshold": "99.99" (type: string)
```

Adding the quotes results in the error `expected 'number' got 'string'`